app/commands/v2/publish.rb

```diff
@@ -153,19 +152,10 @@ def clear_published_items_of_same_locale_and_base_path
         )
       end
 
-      def set_public_updated_at
-        return if edition.public_updated_at.present?
-
-        if update_type == "major" || previous_item.blank?
-          edition.update_attributes!(public_updated_at: default_datetime)
-        else
-          edition.update_attributes!(public_updated_at: previous_item.public_updated_at)
-        end
-      end
-
-      def set_first_published_at
-        return if edition.first_published_at.present?
-        edition.update_attributes!(first_published_at: default_datetime)
+      def set_timestamps
```

Comment:
If this were to happen after `set_update_type` it would mean we can avoid having to pass in the `update_type`, which I think would be nice for when we get rid of it.

Reply:
Good shout 👍